packages/cannon-worker-api/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@pmndrs/cannon-worker-api",
+  "version": "0.0.1",
+  "description": "web worker api used in @react-three/cannon",
```

This is the package that you want to use in other projects.

```suggestion
  "description": "cannon-es web worker api",
```

---

Updated the description now 👍